README.md

```diff
@@ -60,9 +60,9 @@ For a more detailed explanation on how things work, check out [Nuxt.js docs](htt
 
 ## Disclaimer
 
-The Low Impact Website code is highly customized to the Organic Basics' website setup, and will more than likely not easily transfer to your setup.
+The Low Impact Website code is highly customized to the Organic Basics website setup, and will more than likely not easily transfer to your setup.
 
-Thus the main outcomes of making this code available to the public will likely be in sharing learnings and techniques, instead of providing a plug-and-play solution for low impact websites. But if you want to make such a thing, let us know! 
+Thus the main outcomes of making this code available to the public will likely be in sharing learnings and techniques, instead of providing a plug-and-play solution for low impact websites. But if you want to do such a thing, let us know! 
```

Comment:
I don't see a meaningful difference between using "make" and "do" here. I think "make" is nicer, since it implies a built solution more than "do".